### ISSUE TITLE
PPTs failing accessibility test: Missing Table Header

### ISSUE THREAD

**Issue by bob-dlc** [gitbrent/PptxGenJS#1299]
The accessibility checker built into PowerPoint reports that tables are "Missing Table Header" even when I set options to {autoPage:true, autoPageRepeatHeader:true}. The top row of the table indeed acts like a header _visually_, but not _semantically_ as far as PowerPoint is concerned. As such, the PPTs fail accessibility testing and are inaccessible for users of assistive technologies such as screen readers. Are there workarounds and/or can this be fixed? Thanks.
